### ISSUE TITLE
ProjectGuid & SDK-based projects

### ISSUE THREAD

**Issue by 3F** [3F/vsSolutionBuildEvent#40]
For SDK-based projects the `$(ProjectGuid)` does not exist anymore, i.e. this property is empty by default.

Possible problem:
```
[ERROR]: Failed getting project from EnvDTE: An equivalent project (a project with the same global properties and tools version)
is already present in the project collection, with the path "<path_to_project_file>". 
To load an equivalent into this project collection, unload this project first.
```
related: https://github.com/3F/DllExport/issues/36#issuecomment-320794498

Some information is still can be received from `CurrentSolutionConfigurationContents`, but...

**Comment by 3F** [comment]
> An equivalent project is already present in the project collection

After #53 we shouldn't have this problem anymore. An equivalent projects will be avoided by the new logic that was based on MvsSln:

https://github.com/3F/vsSolutionBuildEvent/blob/f4c14dc7101b142e9965326072c458ee44c9f891/vsSolutionBuildEvent/Environment.cs#L72-L73

https://github.com/3F/vsSolutionBuildEvent/blob/f4c14dc7101b142e9965326072c458ee44c9f891/vsSolutionBuildEvent/Environment.cs#L317-L319

However, `ProjectGuid` behavior still is addressed to MvsSln as an actual problem because of obsolete MS feature. But this is another story: https://github.com/3F/MvsSln/issues/18
